test/checks/navigation/skip-link.js

```diff
@@ -7,31 +7,26 @@ describe('skip-link', function() {
 		fixture.innerHTML = '';
 	});
 
-	it('should return false if the href points to another document', function() {
-		fixture.innerHTML =
-			'<a href="something.html#mainheader">Click Here</a><h1 id="mainheader">Introduction</h1>';
-		var node = fixture.querySelector('a');
-		assert.isFalse(checks['skip-link'].evaluate(node));
-	});
-
-	it('should return false if the href points to a non-existent element', function() {
-		fixture.innerHTML =
-			'<a href="#spacecamp">Click Here</a><h1 id="mainheader">Introduction</h1>';
-		var node = fixture.querySelector('a');
-		assert.isFalse(checks['skip-link'].evaluate(node));
-	});
-
 	it('should return true if the href points to an element with an ID', function() {
 		fixture.innerHTML =
 			'<a href="#target">Click Here</a><h1 id="target">Introduction</h1>';
+		axe._tree = axe.utils.getFlattenedTree(fixture);
 		var node = fixture.querySelector('a');
-		assert.isTrue(checks['skip-link'].evaluate(node));
+		assert.isTrue(axe.commons.dom.isSkipLink(node));
```

Comment:
Why are you running a commons here, and not the check?

Reply:
I think I had moved them to the `isSkipLink` test before deciding to move it back, but forgot to change it back to a check test. Good catch.